csharp/src/Ice/Ice1Parser.cs

```diff
@@ -178,6 +178,7 @@ internal static Identity ParseIdentity(string s)
         internal static (Identity Identity,
                          string Facet,
                          InvocationMode InvocationMode,
+                         TimeSpan? invocationTimeout,
```

Comment:
I would not add new proxy options to the ice1 format. The ice1 format's main job is compatibility / interop with Ice 3.7 - if you don't care about that, you should use ice2 and URIs.

Reply:
removed Ice1 option